src/services/context_menu/context_menu_service.cpp

```diff
@@ -198,8 +198,7 @@ namespace big
 		{
 			double distance = 1;
 
-			const auto get_closest_to_center = [this, &distance](auto entity_getter_func) -> auto
-			{
+			const auto get_closest_to_center = [this, &distance](auto entity_getter_func) -> auto {
```

Comment:
You can modify the `get_closest_to_center` function to return an entity pointer if it finds one close to the middle of the screen.
Then simplify the code inside the `get_closest_to_center` to not assign to m_veh_pointer, m_ped_pointer, m_prop_pointer but instead do it where you return the entity.